gradle/src/main/groovy/com/flurry/android/symbols/SymbolUploadPlugin.groovy

```diff
@@ -22,11 +22,11 @@ class SymbolUploadPlugin implements Plugin<Project> {
     @Override
     void apply(Project project) {
         UploadProGuardMapping.logger = project.logger
-        getOrCreateConfig(project)
+        getOrCreateConfig project
```

Comment:
Style guide be damned, parenthesis add needed clarity. I don't do it in Ruby or coffeescript and I disagree with removing parens here too.